### PR TITLE
Add a linter hint for astropy-base

### PR DESCRIPTION
**Pull request by Cadair**
~This needs https://github.com/conda-forge/astropy-feedstock/pull/150 to be merged to create all old versions of astropy-base first.~

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/linter_hints/hints.toml [hunk 1]
```diff
@@ -87,3 +87,7 @@ libgfortran-ng = """\
         - {{ compiler("fortran") }}
     ```"""
 openmp = "The `openmp` output has been superseded by `llvm-openmp`."
+astropy = """\
+    Recipes should usually depend on `astropy-base` as opposed to \
+    `astropy`. `astropy-base` only has the required dependancies whereas\
```
Comment (pllim) [inline recipe/linter_hints/hints.toml:92]:
```suggestion
    `astropy`. `astropy-base` only has the required dependancies whereas \
```

**Comment by beckermr** [comment]
@Cadair did you add a hint to the pinnings repo yet?

**Comment by Cadair** [comment]
Isn't that what this is? I am not sure :laughing:

**Comment by beckermr** [comment]
Oh shoot wrong PR comment. 🤦

**Pull request merged**